cpp/lib/include/FileUtil.h

```diff
@@ -127,7 +127,7 @@ class Directory {
     public:
         Entry(const Entry &other);
 
-        /** \return the full name of the entry, i.e., including the directory path. */
+        /** \return the name of the entry w/o the directory path. */
```

Comment:
Then it might be good if we had a GetFullName() function, so we do not need to build the full name again by hand.

Reply:
It exist now.